src/twisted/conch/scripts/ckeygen.py

```diff
@@ -202,10 +203,18 @@ def _defaultPrivateKeySubtype(keyType):
         return "PEM"
 
 
-def printFingerprint(options):
+def getKeyOrDefault(options):
```

Comment:
Do we need it as a public API.

----

Public API or private API, I think that this needs a docstring.


---------

I know that `printFingerprint` is public and without documentation, but I think that for the new code we should try to reduce the public interface, and add documentation.

and mabye also add type hinting?

Reply:
No, this really doesn't need to be public. Agreed, it could use a docstring. Updated.